packages/block-editor/src/components/iframe/use-scale-canvas.js

```diff
@@ -360,12 +332,12 @@ export function useScaleCanvas( {
 			} );
 		}
 
-		if ( scaleValue < 1 ) {
+		if ( isZoomedOut ) {
```

Comment:
I note the original conditional was `scaleValue < 1` and now the conditional is `scale !== 1`.

Was that intentional?

Reply:
We can't check `scale < 1` directly because `scale` could be the string `auto-scaled`. 

The intention for this line before was to check for `isZoomedOut`, but we didn't want the `useEffect` to run when that changed. So, we used `scaleValue` and didn't notice the checks were different. I think using `isZoomedOut` here is better than the check before since it will always be the same result instead of having two different checks for the same thing.